### PR TITLE
Network reserved ips as simple int field

### PR DESCRIPTION
**Pull request by mkurek**
POC of using plain int field to mark some ips as reserved (in network & dhcp) instead of creating instance of the ip with reserved status.

### CODE REVIEW

File: src/ralph/ralph2_sync/subscribers.py [hunk 1]
```diff
@@ -499,13 +500,13 @@ def sync_network_to_ralph3(data):
...
     net.vlan = data['vlan']
     net.dhcp_broadcast = data['dhcp_broadcast']
-    if data['reserved_ips']:
-        for address in data['reserved_ips']:
-            ip = IPAddress.objects.get_or_create(
-                address=address,
-                defaults=dict(status=IPAddressStatus.reserved)
-            )[0]
-            ip.save()  # trigger save to reassign to proper network
+    # if data['reserved_ips']:
```
Comment (ghost) [inline src/ralph/ralph2_sync/subscribers.py]: could we delete this commented block?

File: src/ralph/networks/models/networks.py [hunk 2]
```diff
@@ -314,6 +312,8 @@ def save(self, *args, **kwargs):
             >>> network.min_ip, network.max_ip, network.gateway
             (3232235776, 3232236031, None)
         """
+        # TODO: gateway status? reserved? or maybe regular ip field instead of
+        # PK?
         if self.gateway_id and not self.gateway.is_gateway:
```
Comment (ghost) [inline src/ralph/networks/models/networks.py:362]: `self.gateway.is_gateway`?

Comment (ar4s) [inline src/ralph/networks/models/networks.py:362]: but what if `self.gateway` will be `None`?

**Comment by ghost** [comment]
:ok: to me maybe excluding the migration code which is untested :)

**Comment by ar4s** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**